memorious/worker.py

```diff
@@ -16,6 +16,9 @@ def boot(self):
                                         limit=1)
 
     def periodic(self):
+        for crawler in manager:
```

Comment:
This would be run for every crawlers after every task. Would it make sense to introduce an hourly rate limit here like we have in the aleph worker so we only do crawler-killing every now and then?